File: ProEventos/Back/src/ProEvents.Domain/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEvents.Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? EventDate { get; set; }
        public string Theme { get; set; }
        public int AmountPeople { get; set; }        
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Batch> Batch { get; set; }
        public IEnumerable<SocialNetwork> SocialMedia { get; set; }
        public IEnumerable<EventSpeaker> EventSpeakers { get; set; }

    }
}

File: ProEventos/Front/ProEvents-App/src/app/models/Event.ts
import { Batch } from "./Batch";
import { SocialNetwork } from "./SocialNetwork";
import { Speaker } from "./Speaker";

export interface Event {
  id: number;
  local: string;
  eventDate ?: Date
  theme: string;
  amountPeople: number;
  imageURL: string;
  phone: string;
  email: string;
  batches: Batch[];
  socialMedia: SocialNetwork[];
  eventSpeakers: Speaker[];
}
